src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParseTreeTest.java

```diff
@@ -99,10 +106,13 @@ public void textBeforeJavadocTags()
     @Test
     public void customJavadocTags()
             throws IOException {
-        final String filename = getDocPath("InputCustomJavadocTags.txt");
-        final ParseTree generatedTree = parseJavadoc(getFileContent(new File(filename)));
-        final ParseTree expectedTree = ParseTreeBuilder.treeCustomJavadocTags();
-        compareTrees(expectedTree, generatedTree);
+        final String inputFilename = getDocPath("InputCustomJavadocTags.txt");
+        final String outputFilename = getOutputDocPath("OutputCustomJavadocTags.txt");
+        final String expectedTree = getFileContent(new File(outputFilename))
+                .replaceAll("\\\\r\\\\n", "\\\\n");
+        final String generatedTree = printFileAst(new File(inputFilename))
+                .replaceAll("\\\\r\\\\n", "\\\\n");
```

Comment:
imo, this should be a common method, [like in `BaseCheckTestSupport` like we do for Java grammar](https://github.com/checkstyle/checkstyle/blob/0fcbcf36a1ac07ef74d3e2df0b7603a5be102f91/src/test/java/com/puppycrawl/tools/checkstyle/BaseCheckTestSupport.java#L155).
We already know one other class will be using these methods.